specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2025-03-03/gallery.json

```diff
@@ -1432,36 +1373,50 @@
             }
           }
         },
+        "x-ms-long-running-operation": true,
         "x-ms-examples": {
-          "List galleries in a resource group.": {
-            "$ref": "./examples/galleryExamples/Gallery_ListByResourceGroup.json"
+          "Create or update a simple gallery Script.": {
+            "$ref": "./examples/galleryExamples/GalleryScript_Create.json"
           }
-        },
-        "x-ms-pageable": {
-          "nextLinkName": "nextLink"
         }
-      }
-    },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute/galleries": {
-      "get": {
+      },
+      "patch": {
         "tags": [
-          "Galleries"
+          "GalleryScripts"
         ],
-        "operationId": "Galleries_List",
-        "description": "List galleries under a subscription.",
+        "operationId": "GalleryScripts_Update",
+        "description": "Update a gallery Script Definition.",
         "parameters": [
           {
             "$ref": "../../../common-types/v1/common.json#/parameters/SubscriptionIdParameter"
           },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/GalleryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/GalleryScriptNameParameter"
+          },
           {
             "$ref": "../../../common-types/v1/common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "galleryScript",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GalleryScriptUpdate"
+            },
```

Comment:
>  "200": { [](http://example.com/codeflow?start=8&length=10)

[ARMBlockingComment] Status code 202 is required for long running patch operations